src/main/java/code/practice/tasks/StreamTasks.java

```diff
@@ -0,0 +1,55 @@
+package code.practice.tasks;
+
+import code.practice.exceptions.UnexpectedMajorException;
+
+import java.util.List;
+import java.util.stream.IntStream;
+
+public class StreamTasks {
+    //Given list of strings representing int numbers. Return sum of all positive elements. write each element to console.
+    public int sumOfPositiveIntegers(List<String> numbers) {
+        return numbers.stream()
+                .mapToInt(Integer::parseInt)
+                .filter(e -> e >= 0)
+                .peek(System.out::println)
+                .sum();
+    }
+
+    //Given list of lists of strings representing int numbers. Return sum of all positive elements.
+    public int sumOfIntegers(List<List<String>> numbersLists) {
+        return numbersLists.stream()
+                .flatMap(List::stream)
+                .mapToInt(Integer::parseInt)
+                .filter(e -> e >= 0)
+                .sum();
+    }
+
+    //Print sequence between from and to
+    public void justPrintSequence(int from, int to) {
+        IntStream
+                .rangeClosed(from, to)
+                .forEach(System.out::println);
+    }
+
+    //Return any negative int string from the given list of ints. Throw some exception if not found.
+    public String getAnyNegativeIntAsString(List<Integer> listOfInts) throws UnexpectedMajorException {
+        try {
+            return listOfInts.stream()
+                    .filter(e -> e < 0)
+                    .map(Object::toString)
+                    .findAny()
+                    .orElse("");
+        } catch (Exception e) {
+            throw new UnexpectedMajorException("Caught UnexpectedMajorException: " + e.getMessage());
```

Comment:
You need to throw exception if given list doesn't contain any negative ints. No need to catch exceptions.